src/test/java/uk/gov/companieshouse/web/pps/service/finance/impl/FinanceServiceHealthCheckImplTest.java

```diff
@@ -240,4 +240,29 @@ void healthCheckOtherExceptionPhasingTime() throws Exception {
         assertFalse( result.getBaseModelAttributes().isPresent());
 
     }
+
+    @Test
+    @DisplayName("Test timezone conversation for BST against GMT/UTC")
+    void testTimezoneConversionForBST() throws Exception {
+        FinanceHealthcheck mockFinancialHealthCheck = new FinanceHealthcheck();
+
+        mockFinancialHealthCheck.setMessage(FinanceHealthcheckStatus.UNHEALTHY_PLANNED_MAINTENANCE.getStatus());
+        mockFinancialHealthCheck.setMaintenanceEndTime(MAINTENANCE_END_TIME);
+
+        when(mockPenaltyPaymentService.checkFinanceSystemAvailableTime()).thenReturn(mockFinancialHealthCheck);
+        when(mockPenaltyConfigurationProperties.getSignOutPath()).thenReturn(SIGN_OUT_PATH);
+
+        var result = financeServiceHealthCheck.checkIfAvailable();
+
+        assertTrue( result.getUrl().isPresent());
+        assertEquals(SERVICE_UNAVAILABLE_VIEW_NAME, result.getUrl().get());
+
+        assertTrue(result.getModelAttributes().isPresent());
+        assertTrue(result.getModelAttributes().get().containsKey(DATE_STR));
+
+        String displayDateFormat = (String) result.getModelAttributes().get().get(DATE_STR);
+        assertTrue(displayDateFormat.contains("5:05 am"));
```

Comment:
We seem to be losing the minute from the input date-time string?
I think it would be better to use assertEquals here like

```suggestion
        assertEquals("5:05 am on Friday 3 August 2001", displayDateFormat);
```